yarn.lock

```diff
@@ -5924,9 +6917,10 @@ lru-cache@^5.0.0, lru-cache@^5.1.1:
   dependencies:
     yallist "^3.0.2"
 
-lux-design-system@^2.0.0:
-  version "2.0.3"
-  resolved "https://registry.yarnpkg.com/lux-design-system/-/lux-design-system-2.0.3.tgz#1a2f646788a807514e392230ae403371120f4bfb"
+lux-design-system@2.0.0:
+  version "2.0.0"
```

Comment:
Downgrading by these patch releases broke the Jest test suites: https://circleci.com/gh/pulibrary/figgy/22944?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

I can look to try and resolve this on a separate branch

Reply:
I'm updating the snapshots.

Reply:
the test will fail though `expect(wrapper.vm.labelerOpts.unitLabel).toEqual('f. ')`

Reply:
Is this due to bugs having been resolved in the last three patch releases?

Reply:
@jrgriffiniii I dont mind if you want to resolve the jest test in a separate branch

Reply:
we can try to keep the lux-design-system at 2.13. version but we will have to upgrade the node version. this is why I downgraded. circleci complained about the node we have vs the node lux expects.

Reply:
https://circleci.com/gh/pulibrary/figgy/22968 builds with the latest release of LUX, but I'm still getting Jest failures (I'm working to determine why https://github.com/pulibrary/figgy/blob/master/app/javascript/test/orderManagerFilesetsForm.spec.js#L109 does not set the `method`)

Reply:
https://vue-test-utils.vuejs.org/guides/#testing-key-mouse-and-other-dom-events does feature this pattern as an example, so I'm going to assume that this might be a typing error.

Reply:
https://github.com/pulibrary/figgy/compare/upgrade-webpacker-jrgriffiniii#diff-04d4a4d686979eba06a39bb6f26ad4a1R116 and https://github.com/pulibrary/figgy/compare/upgrade-webpacker-jrgriffiniii#diff-04d4a4d686979eba06a39bb6f26ad4a1R128 were necessary to ensure that the component functions as expected, but I'm fairly certain that this is a Vue anti-pattern

Reply:
thank you @jrgriffiniii